workflow/pred_temperature_ridge.ipynb

```diff
@@ -425,58 +468,675 @@
     ")\n",
     "\n",
     "# fit preprocessor with training data\n",
-    "preprocessor.fit(\n",
-    "    precursor_field.sel(\n",
-    "        time=slice(str(start_year), str(start_year + train_samples - 1))\n",
-    "    )\n",
-    ")"
+    "preprocessor.fit(train_slice)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 61,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# preprocess the whole precursor field\n",
-    "precursor_field_prep = preprocessor.transform(precursor_field)"
+    "train_prep = preprocessor.transform(train_slice)"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Resample data to the calendar"
+    "### 6 - Resample data to the calendar"
```

Comment:
You can use xarray.set_option with the `display_expand_data` setting to hide all the data, and `display_expand_attrs` to hide the attrs by default

Also, using the html view could be nicer.

Reply:
Handy tip thanks